temporian/core/data/event.py

```diff
@@ -22,6 +22,27 @@
 
 
 class Event(object):
```

Comment:
Nice text. I tried some improvements (with the help of chatgpt) :):

An *event* represents the structure, or schema, of a collection of indexed multivariate time series. An event does not contain any actual data, but is instead used as a reference to describe the format of the input, intermediate results, or output of a *Processor* (i.e., a computation graph).

Informally, an event defines the name and data types of each time series, as well as the key and data type of the index (if any).

There are several ways to create an event:

- When importing data from a Pandas DataFrame with the `a = NumpyEvent.from_dataframe(...)` method, the event is automatically constructed and available with the `a.schema()` method.
- Manually using the `temporian.input_event(...)` method to specify the name and data types of each time series and the key and data type of the index.
- (Not recommended) By instantiating the Event class directly.